article/background.tex

```diff
@@ -4,107 +4,69 @@ \section{Background}
-Lastly, distributing code in virtual machine images allows the original authors to solve dependency problems that arise from imperfect package managers, imperfect repositories, and constant package updates; instead the authors distribute a locked snapshot of a working system.
-
-System virtualization offers a way to portably freeze and preserve environments, but are limited due to the size of the "full disk images".
-Additionally VM's must be be "booted" which can be costly if many instances are needed.
-Modern advances in container technology have allowed similar benefits but strip redundancy by making limited use of the host machine, specifically the hypervisor.
-Containers enable a complete working environment as small as a few Megabytes, and can be started as quickly as a normal process.
-Many container images are publicly available via public image repositories.
-
-Containers technology is not a recent invention, but the term "container" gained popularity alongside the Docker toolset.
-Over time Docker and other organizations have come together under a Linux Foundation project, the "Open Container Initiative" (OCI).
-The OCI governing body has produced an open specification for containers, which can be used by various container runtimes and toolsets.
-OCI complient container images in most cases can be executed identically with Docker, Podman, or other OCI compliant tools.
-
-While OCI images are nearly ubiquitous in the industry, Singularity (recently renamed to Apptainer) is a toolset that was developed specifically for High Performance Computing.
-Singularity has support for converting OCI images into singularity images, and recent versions of Apptainer have also added support to natively run OCI containers.
-Podman apears to be gaining traction in the HPC community, but Apptainer is still required on many systems.
-
-One of the most significant downsides to using Docker in HPC environments was that it required root privilages.
-However, recent advances in container technology have made this unnecessary, and it is now considered best practice to run containers without root privilages when reasonable.
-
-While the original reference OPFVTA article did not leverage this technology, containers can be used improve to improve the reliability and portability of the OPFVTA project.
-In this article, the authors will provide a snapshot of OPFVTA functioning at a certain point in time — mitigating process fragility in view of incrementing software dependency versions.
+Independent verification of published results is a crucial step in the establishment and maintanence of trust in shared scientific understanding. \cite{TODO Too corny to cite Kuhn?}.
+Verification research has become more common due in part to dramatic results \cite{TODO: Salmon}, dedicated communities \cite{TODO: something wholesome}, and widely publicized large-scale reproducibility meta-analyses \cite{RPP}.
```

Comment:
I agree, we should choose a term and make it consistent. I think it will be easier to do that after we make bigger choices though, so I'll leave this comment open.